beets/autotag/mb.py

```diff
@@ -607,6 +607,61 @@ def _parse_id(s: str) -> Optional[str]:
     return None
 
 
+# this was defined within the function below but pep8 made me move it here
+_trans_key = 'transl-tracklisting'
+_is_trans = lambda r: r['type'] == _trans_key and r['direction'] == "backward"
+
+
+def _find_actual_release_from_pseudo_release(pseudo_rel: Dict)\
+      -> Optional[Dict]:
+    relations = pseudo_rel['release']["release-relation-list"]
+
+    # currently we only support trans(liter)ation's
+    actual_id = next(filter(_is_trans, relations), {'target': None})['target']
+
+    if actual_id is None:
+        return None
+
+    return musicbrainzngs.get_release_by_id(actual_id,
+                                            RELEASE_INCLUDES)
+
+
+def _merge_pseudo_and_actual_album(
+        pseudo: beets.autotag.hooks.AlbumInfo,
+        actual: beets.autotag.hooks.AlbumInfo
+        ) -> Optional[beets.autotag.hooks.AlbumInfo]:
+    """
+    Merges a pseudo release with its actual release.
+
+    This implementation is naive, it doesn't overwrite fields,
+    like status or ids.
+
+    According to the ticket PICARD-145, the main release id should be used.
+    But the ticket has been in limbo since over a decade now.
+    It also suggests the introduction of the tag `musicbrainz_pseudoreleaseid`,
+    but as of this field can't be found in any offical Picard docs,
+    hence why we did not implement that for now.
+    """
+    merged = pseudo.copy()
+    merged.update({
+        "media": actual.media,
+        "mediums": actual.mediums,
+        "country": actual.country,
+        "catalognum": actual.catalognum,
+        "year": actual.year,
+        "month": actual.month,
+        "day": actual.day,
+        "original_year": actual.original_year,
+        "original_month": actual.original_month,
+        "original_day": actual.original_day,
+        "label": actual.label,
+        "asin": actual.asin,
+        "style": actual.style,
+        "genre": actual.genre,
+    })
```

Comment:
I don't know if this helps much, be we could reduce some of the typing here by just using a list of fields. That is:

```py
from_actual = {k: actual[k] for k in ["media", "mediums", "country", ...]}
merged.update(from_actual)
```

Reply:
this for syntax is pretty new to me tbh, I've added it to make adding fields less painful